packages/gatsby-image/src/index.js

```diff
@@ -160,6 +185,17 @@ class Image extends React.Component {
       IOSupported = true
     }
 
+    // Chrome Canary 75 added native lazy loading support!
+    // https://addyosmani.com/blog/lazy-loading/
+    if (
+      typeof HTMLImageElement !== `undefined` &&
+      `loading` in HTMLImageElement.prototype
+    ) {
+      // Setting isVisible to true to short circuit our IO code and let the browser do its magic
+      isVisible = true
```

Comment:
maybe to make it clear this is now representing two different states it could be changed to `isVisibleOrNativeLazyLoadingSupported` — verbose is effective :-D

Reply:
@KyleAMathews isn't the `isVisible` state here linked to the image components visibility state for it's picture element? I suggested a refactor [here](https://github.com/gatsbyjs/gatsby/pull/13217#discussion_r273377916), but perhaps that's the wrong approach.